openc3/lib/openc3/packets/structure_item.rb

```diff
@@ -303,6 +303,7 @@ def <=>(other)
     def clone
       item = super()
       item.name = self.name.clone if self.name
+      item.key = self.key.clone if self.key
```

Comment:
Was this an old bug?

Reply:
Sort of.  It makes it a more complete clone.